README.md

```diff
@@ -1,36 +1,45 @@
-=======
-# dojo.io site
+# Dojo.io Site
 
-This project was generated with the [Dojo CLI](https://github.com/dojo/cli) & [Dojo CLI create app command](https://github.com/dojo/cli-create-app).
+Next generation dojo.io site.
 
-## Build
+## Running Dojo.io Locally
```

Comment:
very nitpicky so forgive in advance but think this should be stylised as dojo.io for consistency